index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Tribute Page</title>
+</head>
+<body>
+    <div id="main">
```

Comment:
It's a good idea to use semantic elements on the page whenever possible. Please replace the div tag with a section tag or header tag. always use class attributes for multiple elements instead of id selector

Reply:
Nicely done :+1: